JustLog/Classes/Logger.swift

```diff
@@ -208,12 +213,17 @@ extension Logger: Logging {
     }
     
     internal func log(_ type: LogType, _ message: String, error: NSError?, userInfo: [String : Any]?, _ file: String, _ function: String, _ line: UInt) {
+        
         let messageToLog = logMessage(message, error: error, userInfo: userInfo, file, function, line)
+    
+        let sanitizedMessageToLog = sanitizer(messageToLog, type)
+        
+        let sanitizedMesaage = sanitizer(message, type)
```

Comment:
Why do we call `sanitizer` twice?

Reply:
Good question. I investigated this as part of a ticket: 

Investigation into ‘MessageToLog' and Simple 'Message’ Variable
In line with the following code, the ‘messageToLog’ variable is a more enhanced and detailed string that adds parameters to the simple ‘message’ variable: 

```
if !internalLogger.destinations.isEmpty {

            sendLogMessage(with: type, logMessage: messageToLog, file, function, line)

        } else {

            queuedLogs.append(QueuedLog(type: type, message: mesaage, file: file, function: function, line: line)) //Why message?

        } 
```

```
If there are no logger destinations to choose from, the following sendQueuedLogsIfNeeded() method will be called, detecting that there are queued messages and firing the logs

    private func sendQueuedLogsIfNeeded() {

        if !queuedLogs.isEmpty {

            queuedLogs.forEach { queuedLog in

                sendLogMessage(with: queuedLog.type, logMessage: queuedLog.message, queuedLog.file, queuedLog.function, queuedLog.line)

            }

            queuedLogs.removeAll()

        }

    }
```

Reply:
OK, would it make sense to move these calls to `sanitize` inside of the `if...else` so that we aren't doing double the work for every call?

Reply:
Sure, pushing now.